frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.styled.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "@emotion/styled";
+import type { MouseEventHandler } from "react";
+import { Icon } from "metabase/core/components/Icon";
+import { color } from "metabase/lib/colors";
+import type { AnchorProps } from "metabase/ui";
+import { Anchor } from "metabase/ui";
+
+export const VerticalDivider = styled.div`
+  align-self: stretch;
+  width: 1px;
+  background-color: ${color("border")};
+  padding: 5px 0px;
+`;
+
+export const ActionContainer = styled.div`
```

Comment:
Instead of `div`, we can use `Box` (div), `Group` (flex row), or `Stack` (flex col), so we can take advantage of Mantine's style props. You still might need to use a styled component to add some of the unsupported props, but you can use the theme values here as well.


So for example, with this one we can do:

```
// In your JSX

<ActionContainer p="xs">
	... your stuff goes here ...
</ActionContainer>
```


```
export const ActionContainer = styled(Box)<BoxProps & HTMLAttributes<HTMLDivElement>>`

	&:hover {
		color: ${({theme}) => theme.colors.brand[1]};
	}

`
```

Reply:
Thanks, turns out that component was not used anymore 🙃